meta-ostro/recipes-soletta/soletta/soletta_%.bbappend

```diff
@@ -1,6 +1,6 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
 
-DEPENDS_append = " systemd"
+DEPENDS_append = " systemd libmicrohttpd"
```

Comment:
As far I know there is an open PR there: https://github.com/solettaproject/meta-soletta/pull/48.

We can merge that patch and remove the systemd depends here....